charts/akeyless-gateway/values.yaml

```
@@ -113,6 +113,42 @@ globalConfig:
         cpu: 250m
         memory: 256Mi
 
+cacheHA:
+  enabled: false
+## TODO: Need to find the right value to control the Redis port
+
+  nameOverride: "cache-ha"
+
+  ## In case cacheHA is enabled in the Gateway, and the encryptionKeyExistingSecret parameter has a value
+  ## Akeyless will use this specified encryption key and store it securely within Akeyless Gateway.
+  ## If the encryptionKeyExistingSecret parameter is empty or not specified,
+  ## Akeyless will automatically generate a new encryption key and a new ServiceAccount for K8s.
+  ## for more information: https://docs.akeyless.io/docs/advanced-chart-configuration#cache-configuration
+  ##
+  encryptionKeyExistingSecret:
+  tls:
+    enabled: true
+    authClients: false
+    autoGenerated: true
```

```suggestion
    enabled: false
    authClients: false
    autoGenerated: false
```
why this should be enabled by default?